Traceback (most recent call last):
  File "/home/dan/NeurIPS2023-One-Fits-All/Long-term_Forecasting/main.py", line 128, in <module>
    train_data, train_loader = data_provider(args, 'train')
  File "/home/dan/NeurIPS2023-One-Fits-All/Long-term_Forecasting/data_provider/data_factory.py", line 41, in data_provider
    data_set = Data(
  File "/home/dan/NeurIPS2023-One-Fits-All/Long-term_Forecasting/data_provider/data_loader.py", line 557, in __init__
    self.__read_data__()
  File "/home/dan/NeurIPS2023-One-Fits-All/Long-term_Forecasting/data_provider/data_loader.py", line 574, in __read_data__
    df_0=df_EPS[df_EPS['ticker']==tickers[self.idx]][select_columns]
AttributeError: 'Dataset_EPS' object has no attribute 'idx'